package.json

```diff
@@ -8,11 +8,10 @@
     "start:production": "cross-env NODE_ENV=production node ./output/index.js",
     "start:test": "cross-env NODE_ENV=test MOCK=true node ./output/index.js",
     "start:development": "cross-env NODE_ENV=development node ./output/index.js",
-    "build-server": "babel -d ./output ./src -s",
+    "build-server": "babel -d ./output ./src -s --ignore **/*.test.js",
```

Comment:
These two lines remove the non-necessary files from the final build, correct? 
`--ignore **/*.test.js` and `npm prune --production`

Reply:
Correct, it removes the test files and `devDependencies` npm libraries.